### ISSUE TITLE
Show pubpeer plugin results on experiment pages with publications

### ISSUE THREAD

**Issue by ppavlidis** [PavlidisLab/Gemma#1283]
I don't know how this works, but the [pubpeer browser plugin](https://pubpeer.com/static/extensions) will show a banner at the top of pages that cite a study that has comments. This could be helpful to curators and users. 

We list the pubmed reference (when available), but we must not be displaying it in a way that the pubpeer plugin recognizes, it may need some hinting.

**Comment by ppavlidis** [comment]
I have a feeling it detects DOIs. So I suggest we add the DOI to the information in the citation we display and see if that makes it work. [Here is a dataset](https://gemma.msl.ubc.ca/expressionExperiment/showExpressionExperiment.html?id=24999) where there are [pubpeer comments](https://pubpeer.com/publications/59D4DB0A3F2F6DF3D2FDE0AFD1A87C) on the [paper](https://pubmed.ncbi.nlm.nih.gov/35767995/)

**Comment by arteymix** [comment]
Yes, most likely. I'll adjust the citation format to include a DOI.

**Comment by arteymix** [comment]
It looks like we're simply rendering `BibliographicReference.citation` in the UI. We need to format the citation with the full text URI.

**Comment by oganm** [comment]
looking at the extension code, a `PMID: [PMID]` seems to be sufficient but I was unable to force the extension to see localhost so far so need to test properly

**Comment by oganm** [comment]
Looks like our css doesn't play too well with the addon. The top thing is fine but the inline one isn't. the culprit is `.x-form-item:nth-child(odd)`. trying to find a graceful way to override it

![Image](https://github.com/user-attachments/assets/7cb60190-a112-4d4e-9b6f-7ad85495c353)

**Comment by oganm** [comment]
had to use the class name the addon uses to exlude it but otherwise all is well

![Image](https://github.com/user-attachments/assets/f959e5d1-e5ef-4efa-bfcc-e742763224e9)

**Comment by oganm** [comment]
Fixed in https://github.com/PavlidisLab/Gemma/pull/1339

**Comment by arteymix** [comment]
It will have to be backported in the upcoming 1.31.13 release.

**Comment by oganm** [comment]
https://github.com/PavlidisLab/Gemma/pull/1340
